doc/setup.md

```diff
@@ -10,6 +10,8 @@ how to install them on MacOS and Debian-based systems below.
 - NGINX (v>=1.7)
 - PostgreSQL (v>=9.6)
 - Node.JS/npm (v>=5.8.0)
+- libcurl4-gnutls-dev
```

Comment:
Since I think this comes up mostly on Linux systems, maybe we just discuss it explicitly in that section?